### PR TITLE
[cli/move] Expose --Xdebug flag for sui move disassemble

### PR DESCRIPTION
**Pull request by amnn**
## Description

Replicates a change to the `move` CLI, where the `--Xdebug` flag dumps a compiled module in its most raw form -- the `Debug` format of the `CompiledModule` type in Rust.

This helps to see what's going on in the module's pools which are otherwise hidden in the disassembly output.

## Test Plan

First call to disassemble will forward to the move CLI's `debug` command and print both a source-mapped disassembly, and the raw output, and the second call will print just the raw output:

```
sui$ cargo build
sui$ target/debug/sui move disassemble --Xdebug crates/sui-framework/build/Sui/bytecode_modules/pay.mv
sui$ cp crates/sui-framework/build/Sui/bytecode_modules/pay.mv /tmp/
sui$ target/debug/sui move disassemble --Xdebug /tmp/pay.mv
```

### CODE REVIEW

**Comment by Shuknasty80** [comment]
Nice

File: crates/sui-move/src/disassemble.rs [hunk 1]
```diff
@@ -38,23 +42,31 @@ impl Disassemble {
...
             }
-            .execute(package_path, build_config)?
+            .execute(package_path, build_config)?;
+            return Ok(());
+        }
+
+        // disassembling a bytecode file with no source info
+        assert!(
+            Path::new(&self.module_path).exists(),
+            "Bath path to .mv file"
```
Comment (oxade) [inline crates/sui-move/src/disassemble.rs:54]:
```suggestion
            "Bad path to .mv file"
```

**Review by oxade — APPROVED** [review]
LGTM. Just a nit.

File: crates/sui-move/src/disassemble.rs (hunk 1 above)
Comment (amnn) [reply crates/sui-move/src/disassemble.rs:54]: 😆 good spot

**Pull request merged**